# PD gains (/1000)
Kr: [250.0, 250.0, 250.0]
Komega: [250.0, 250.0, 250.0]

# Inertia, in kgÂ·(m^2) 
Jdiag: [0.01, 0.01, 0.01] #//Diagonal elements of the inertia matrix in body frame

# Arm length, in m. Distance from the motor to the c.o.m of the drone
l: 0.113

# Drag coefficient of the propeller. It is such that the
# moment created by one motor **around the motor axis** is M=cd*f,
# where f is the thrust produced by that motor. M in N.m, and f in N
cd: 0.02

# Thrust curve for single motor, [an ...a1 a0].
# Maps thrust (N) to throttle (normalized [0, 1])
thrust_curve_cw: [0.0, 0.13, 0.0]   # For clockwise props, 2200mAh 4s
thrust_curve_ccw: [0.0, 0.13, 0.0]  # For counterclockwise props, 2200mAh 4s

# Spinning direction of propellers. -1: clockwise, +1: counterclockwise
motor_spin: [-1, 1, -1, 1]

# Center of mass ofter in body frame (m)
com: [0.0, 0.0, 0.0]

mixer: "QUADX"
